pwndbg/ida.py

```diff
@@ -50,14 +52,23 @@
 # to avoid printing the same exception multiple times, we store the last exception here
 _ida_last_exception = None
 
+# to avoid checking the connection multiple times with now delay, we store the last time we checked it
+_ida_last_connection_check = 0
+_IDA_MIN_CONNECTION_DELAY_CHECK = 5
+
 
 @pwndbg.config.Trigger([ida_rpc_host, ida_rpc_port])
```

Comment:
We should probably add here the `ida_timeout` parameter so it will set `socket.setdefaulttimeout` when e.g. user fires `set ida_timeout 5`.

Reply:
Yes, you're right, I added it.